sdk/communication/communication-networktraversal/swagger/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Communication Services Configuration Module
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: azure-communication-network-traversal
+override-client-name: NetworkRelayRestClient
+description: Communication Network Traversal Client
+package-version: 1.0.1
```

Comment:
where has it been determined that this should be 1.0.1? this package is unreleased right? this doesn't match the changelog